.changeset/loud-apricots-fix.md

```diff
@@ -0,0 +1,31 @@
+---
+'@backstage/create-app': patch
+'@backstage/plugin-scaffolder': patch
```

Comment:
This is really a breaking change, since the button will disappear unless app code is change, so let's move the `@backstage/plugins-scaffolder` bump to `minor`.

Reply:
Fixed